graph/telemetry/istio/util/util.go

```diff
@@ -122,3 +122,20 @@ func IsBadDestTelemetry(cluster string, clusterOK bool, svcNs, svc, svcName, wl
 	}
 	return false
 }
+
+// GetQueryScope returns "" if there is no configured queryScope, otherwise queryScope with/without a trailing comma
+func GetQueryScope(prefix, suffix string) string {
```

Comment:
How about adding a unit test for this method in `util_test.go`?

Reply:
BTW, can we pass the query to this method instead of pass prefix and suffix to hide some details? 

If we do not read the codes, it is probably hard to understand the meaning of prefix and suffix. And these two params usually do not change.

like func GetQueryWithScope(query string) string{}, when using this method, just like 
``` golang
query = util.GetQueryWithScope(query)
```

Reply:
good idea, I will push a change.